src/styles/index.scss

```diff
@@ -1,3 +1,136 @@
+:root {
+  --link-color: #00acdc;
+  --color: #616070;
+  --bg-color: #fff;
+  --test-color: #060b35;
```

Comment:
There appears to be a typo in the custom property name `--test-color`. According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), custom properties should be named consistently and clearly. Consider renaming it to something more descriptive, such as `--text-color` if that was the intended purpose.